locales/ca.json

```diff
@@ -0,0 +1,168 @@
+{
+  "lang": "Anglès",
```

Comment:
Quite tricky here. it need to be the name of the language translated.

Reply:
In Transifex, we only see the label to translated and not its ID. Instead, we might have a list of languagues in each locale file, so buttons to choose language are always in currently selected language ?

Reply:
So we have to maintain this list each time we add a new language?

Reply:
Yes. Or instead you can show emoji with country flags if you want to avoid maintaining this. I usually don't show this kind of menu, only rely on browser preferred locales.

Reply:
We still have to maintain theses. There is no way to put a comment or something similar ?

Reply:
I checked and I'm not sure we can do this.

Reply:
There is developer notes here: https://docs.transifex.com/translation/tools-in-the-editor#string-instruction-and-developer-notes

Reply:
OK so I added a comment, I was trying to do it in source language, but should be done in one of the translations. Should be good now.

Reply:
:+1: